haproxy/templates/configmap-ldap-tls.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "haproxy.fullname" . }}"
```

Comment:
This should probably include `-{{ .Values.haproxy.config }}` to include the name suffix?

Reply:
Or just `-ldap-tls` hard coded